soccer/gameplay/role_assignment.py

```diff
@@ -21,6 +21,10 @@ def __init__(self):
 
         # multiply this by the distance between two points to get the cost
         self.position_cost_multiplier = 1.0
+        
+        # A lambda function property that allows customization of cost
+        # Has exactly one parameter, which is a robot
+        self.cost_func = None
```

Comment:
What if we did self.cost_func = lambda r: 0

Reply:
This also gives a bit of an idea about what a function should look like, good for readability